pkg/kubelet/sysctl/safe_sysctls.go

```diff
@@ -77,15 +71,15 @@ func SafeSysctlAllowlist() []string {
 	if goruntime.GOOS != "linux" {
 		return nil
 	}
-	return getSafeSysctlAllowlist(getKernelVersion)
-}
-
-func getSafeSysctlAllowlist(getVersion func() (*version.Version, error)) []string {
-	kernelVersion, err := getVersion()
+	kernelVersion, err := utilkernel.GetVersion()
```

Comment:
@cyclinder 's version allows unit testing that the code does the expected thing when the `getVersion` call returns an error too...